src/test/java/ru/ayaz/CommentTest.java

```diff
@@ -0,0 +1,49 @@
+package ru.ayaz;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.mock;
+
+public class CommentTest {
+
+    @Test
+    public void shouldBeEqualWhenReferencesAreSame() {
```

Comment:
This test appears when i code with strict TDD style. In the first step i have equals method with "return false;" statement only. Check that test fails. Then i add reference check. Check that test succeed and then write new tests.

Ah, i should've refactored my code and remove reference checking... And only then i can remove this test. If i will just remove test i leave code uncovered and loose 100% code coverage:) 

Agree